Earthfile

```diff
@@ -368,22 +365,14 @@ uki-tools-image:
 # This is for easy testing SecureBoot locally for development purposes
 # Installing this keys in other place than a VM for testing SecureBoot is irresponsible
 uki:
-    FROM ubuntu
-
-    ARG TARGETARCH
-    COPY +version/VERSION ./
-    RUN echo "version ${VERSION}"
-
-    ARG --required FAMILY # The dockerfile to use
-    ARG --required FLAVOR # The distribution E.g. "ubuntu"
-    ARG --required FLAVOR_RELEASE # The distribution release/version E.g. "20.04"
-    ARG --required VARIANT
-    ARG --required MODEL
-    ARG --required BASE_IMAGE
+    # Uncomment if ISO_NAME is needed (the code that uses it is commented out below)
```

Comment:
not needed anymore, we used to provide the full anme for the artifacts but we do that later in other target :+1: